### ISSUE TITLE
Django Project Layout - config/ location

### ISSUE THREAD

**Issue by ziadrahhal** [twoscoops/two-scoops-of-django-1.11#49]
Page 26

* Chapter: 3
* Section: 3.3
* Subsection:

## Description 
config/ is not placed inside the <django_project_root> according to Example 3.4, however, on page 26 it is written that config/ must be inside the <django_project_root>
So where should it be? inside the <repositoy_root> at the same directory level as the <django_project_root> or inside the <django_project_root>?

> If grammar, enter problematic text here, otherwise delete this line.

```
If code-related, enter it within these backticks. Otherwise, delete this segment.
```

**Comment by jmwoloso** [comment]
+1 that's why I'm here as well.

**Comment by jmwoloso** [comment]
see: #32

**Comment by pydanny** [comment]
Closed as a duplicate of #32
